commands/alpha/rollouts/advance/advance.go

```diff
@@ -65,7 +65,9 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	rolloutName := args[0]
 	waveName := args[1]
 
-	rollout, err := rlc.Get(r.ctx, rolloutName)
+	// TODO(droot): plumb the namespace value from commandline flags
```

Comment:
The `kpt live` code already have functionality for this. Presumably we want to use the default value from the kubeconfig and allowing the user to override this with the `--namespace` flag.

Reply:
Great, we can use whatever existing commands are using.

Reply:
Added a good first issue as well for this: https://github.com/GoogleContainerTools/kpt/issues/3814